spyglass/src/main/java/com/linkedin/android/spyglass/tokenization/impl/WordTokenizer.java

```diff
@@ -411,14 +422,7 @@ protected int getSearchEndIndex(final @NonNull Spanned text, int cursor) {
         }
 
         // Get index of the start of the first span after the cursor (or text.length() if does not exist)
-        MentionSpan[] spans = text.getSpans(0, text.length(), MentionSpan.class);
-        int closestAfterCursor = text.length();
-        for (MentionSpan span : spans) {
-            int start = text.getSpanStart(span);
-            if (start < closestAfterCursor && start >= cursor) {
-                closestAfterCursor = start;
-            }
-        }
```

Comment:
This is nifty, I didn't know about the `nextSpanTransition(..)` method when I wrote this many years ago, very nice! 👍